### PR TITLE
Some further polishing, more to come

### PR DESCRIPTION
**Pull request by yveskerwyn**

### CODE REVIEW

File: projorg_repos.md [hunk 1]
```diff
@@ -1,34 +1,51 @@
...
     - org_support (all support requests (tickets) which are not in project repo yet)
-    - org_internalit (internal IT related & organization of company)
+    - org_internalit (internal IT & organization of company)
     - org_product (product management)
     - org_marketing
     - org_finance
     - org_legal
     - org_hr 
-    - org_quality (all QA related issues and automation code, performance testing, portal testing and automated tests)
+    - org_quality (all Q&A related issues, automation code, performance testing, portal testing and automated tests)
```
Comment (FastGeert) [inline projorg_repos.md]: It is Quality Assurance, not Questions & Answers

File: prodorg_repos.md [hunk 2]
```diff
@@ -17,15 +17,17 @@ Below each of the "prodorg" repositories is discussed.
...
+  - Starting point for people to find their way in all other repos in the prodorg
+  - Holds specifications and product requirements document (PRD) information common to the repos in the prodorg
 - Milestones
   - Are releases (no dates) for a product, used to group work for coding, testing, ...
 - Types of issues in the ``home`` repo:
-  - Only issues of type Feature Request (FR)  
-- This is the ideal repo to define your roadmap covering multiple product and component repo's in the prodorg
+  - Only issues of type Question
+    - Used for questions that are relevant to the whole product organization
+    - All other questions should be asked in the repository the question relates too
```
Comment (FastGeert) [inline prodorg_repos.md]: A roadmap without FR ?

File: stories.md [hunk 3]
```diff
@@ -1,103 +1,103 @@
...
 
-- describe what story is about
+Describe what story is about.
 
-## REQUIREMENTS
+## LINKS:
 
-- Add specifications, remarks, implementation hints, requirements
-- IMPORTANT
+- Tasks: link to a Kanban board, using waffle.io
```
Comment (FastGeert) [inline stories.md]: link to kanban of story:
https://waffle.io/gig-projects/org_development?search=js8beta:&label=type_task

Comment (yveskerwyn) [inline stories.md]: OK added

File: prodorg_repos.md (hunk 2 above)
Comment (yveskerwyn) [inline prodorg_repos.md]: I added feature request too

**Pull request merged**